tests/providers/google/cloud/sensors/test_bigquery.py

```diff
@@ -37,7 +37,7 @@
 TEST_IMPERSONATION_CHAIN = ["ACCOUNT_1", "ACCOUNT_2", "ACCOUNT_3"]
 
 
-class TestBigqueryTableExistenceSensor(TestCase):
+class TestBigqueryTableExistenceSensor:
```

Comment:
If we're switching from unittest to pytest then why are we even using test classes at all? We might as well go all the way and make the tests into functions. I'd rather make this change now while we're already modifying the code.

Reply:
I hope that `classes` vs `functions` it is not new `tabs` vs `spaces` 🤣 
Personally I do not have any preferences, outside of Airflow I use both of them depends on situation.

If module contain a lot of different functions / classes which need to be tested  then classes very nice to use as namespace so developers could easily found tests for specific part, like:

```python

class TestAwesomeHook:

    def test_init(self, ...):
        ...

    def test_init_failure_reason_1(self, ...):
        ...

    def test_get_conn(self, ...):
        ...

class TestAnotherAwesomeHook:

    def test_init(self, ...):
        ...

    def test_init_failure_reason_1(self, ...):
        ...

    def test_get_conn(self, ...):
        ...
```

vs

```python

def test_awesome_hook_init(...):
    ...

def test_awesome_hook_init_failure_reason_1(...):
    ...

def test_another_awesome_hook_init(...):
    ...

def test_another_awesome_hook_get_conn(...):
   ...

def test_awesome_hook_init_failure_reason_1(...):
    ...

def test_awesome_hook_get_conn(...):
   ...

```

Reply:
I agree! Tried retaking a look, and in my opinion as well, having class names for sensors makes the tests lot more readable. @dimberman I would request you to re-evaluate your request to remove classes. Thanks